storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -322,6 +324,12 @@ static struct my_option my_long_options[] = {
      &opt_fifo_timeout, &opt_fifo_timeout, 0, GET_INT, REQUIRED_ARG, 60, 1,
      INT_MAX, 0, 0, 0},
 
+    {"timeout", OPT_TIMEOUT,
```

Comment:
If xtrabackup waits for a backup lock for 120 seconds, would this timeout cause a failure?

Reply:
No, this timeout controls the time we wait on epoll for a reply on a socket we sent data.
Xtrabackup been waiting on locks will not interfere here.